pgmpy/readwrite/XDSL.py

```diff
@@ -314,6 +319,10 @@ def get_variables(self):
         nodes_elem = etree.SubElement(self.root, "nodes")
 
         for var in self.model.nodes:
+            if isinstance(var, str) and " " in var:
+                raise UserWarning(

```

Comment:
@Nimish-4 We should use `pgmpy.global_vars.logger` here. This would then be `logger.warning( ....)`